Task6/src/main/java/arrays/ComparableString.java

```diff
@@ -0,0 +1,42 @@
+package arrays;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Random;
+
+public class ComparableString {
+    ArrayList<String> arrayList = new ArrayList<String>();
```

Comment:
This array list should be created at App main method.
ComparableString class should have only useful methods.

Reply:
done